modules/administration-guide/partials/proc_authenticating-to-the-che-server-using-openid.adoc

```diff
@@ -1,109 +0,0 @@
-
-[NOTE]
-====
-.Providing user-specific data
-
-Since {identity-provider} may store user-specific information (first and last name, phone number, job title), there is a special implementation of the *ProfileDao* that can provide this data to consumers. The implementation is read-only, so users cannot perform create and update operations.
-====
-
-
-[id="obtaining-the-token-from-keycloak_{context}"]
-== Obtaining the token from credentials through {identity-provider}
-
-Clients that cannot run JavaScript or other clients (such as command-line clients or Selenium tests) must request the authorization token directly from {identity-provider}.
-
-To obtain the token, send a request to the token endpoint with the username and password credentials. This request can be schematically described as the following cURL request:
-
-[subs="+quotes,+attributes"]
-----
-$ curl --insecure --data "grant_type=password&client_id={prod-deployment}-public&username=__<USERNAME>__&password=__<PASSWORD>__" \ <1> <2>
-https://<keyckloak_host>/auth/realms/{prod-deployment}/protocol/openid-connect/token <3>
-----
-<1> {prod} username
-<2> {prod} user's password
-<3> {identity-provider} host
-
-The {prod-short} dashboard uses a customized {identity-provider} login page and an authentication mechanism based on `grant_type=authorization_code`. It is a two-step authentication process:
-
-. Logging in and obtaining the authorization code.
-. Obtaining the token using this authorization code.
-
-[id="obtaining-the-token-from-openshift-token-through-keycloak_{context}"]
-== Obtaining the token from the OpenShift token through {identity-provider}
-
-When {prod-short} was installed on OpenShift using the Operator, and the OpenShift OAuth integration is enabled, as it is by default,
-the user's {prod-short} authentication token can be retrieved from the user's OpenShift token.
-
-To retrieve the authentication token from the OpenShift token, send a schematically described cURL request to the OpenShift token endpoint:
-
-[subs="+quotes,+attributes"]
-----
-$ curl --insecure -X POST  \
--d "client_id={prod-deployment}-public" \
--d "subject_token=__<USER_OPENSHIFT_TOKEN>__" \ <1>
--d "subject_issuer=__<OPENSHIFT_IDENTITY_PROVIDER_NAME>__" \ <2>
---data-urlencode "grant_type=urn:ietf:params:oauth:grant-type:token-exchange" \
---data-urlencode "subject_token_type=urn:ietf:params:oauth:token-type:access_token" \
-https://__<KEYCKLOAK_HOST>__/auth/realms/{prod-deployment}/protocol/openid-connect/token <3>
-----
-<1> The token retrieved by the end-user with the command `oc whoami --show-token`
-<2> `openshift-v4` for OpenShift 4.x and `openshift-v3` for OpenShift 3.11
-<3> {identity-provider} host
-
```

Comment:
cc: @tolusha I thought this was one of the top common question  - how to retrieve the keycloak token based on OpenShift token.

@xbaran4 what is the reason behind the removal of this section?

Reply:
https://github.com/eclipse/che/issues/17266
TLDR; @skabashnyuk thinks that these section are more helpful for Che developers rather than admins and they will be moved closer to the code itself. https://github.com/eclipse/che/pull/19285